jaeger-spark-dependencies-common/src/main/java/io/jaegertracing/spark/dependencies/model/Process.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.spark.dependencies.model;
+
+import java.io.Serializable;
+
+/**
+ * @author Pavol Loffay
+ */
+public class Process implements Serializable {
```

Comment:
Wondering whether it would be better to have the java model is some independent package, as it could be generally useful for Java tooling wanting to read data from the jaeger storage?

Reply:
agree, I would like to raise some issues about generic tests and reusable parts in java.